http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala

```diff
@@ -502,20 +507,6 @@ object RuntimeRoutes {
 
 }
 
-final case class CreateRuntime2Request(labels: LabelMap,
-                                       jupyterUserScriptUri: Option[UserScriptPath],
-                                       jupyterStartUserScriptUri: Option[UserScriptPath],
-                                       runtimeConfig: Option[RuntimeConfigRequest],
-                                       userJupyterExtensionConfig: Option[UserJupyterExtensionConfig],
-                                       autopause: Option[Boolean],
-                                       autopauseThreshold: Option[FiniteDuration],
-                                       defaultClientId: Option[String],
-                                       toolDockerImage: Option[ContainerImage],
-                                       welderDockerImage: Option[ContainerImage],
-                                       scopes: Set[String],
-                                       customEnvironmentVariables: Map[String, String],
-                                       diskConfig: Option[DiskConfigRequest])
-
 sealed trait UpdateRuntimeConfigRequest extends Product with Serializable {
```

Comment:
why not move these as well?

Reply:
moving these introduces lots of diffs. given the size of this PR, I don't feel like moving these I don't need in this PR